### PR TITLE
Upgrade TodoListSPA to msal-angular-v2

### PR DESCRIPTION
**Pull request by derisen**
This PR:

- Replaces the TodoListSPA Angular 9 project using msal-angular-v1 with an Angular 11 project using msal-angular-v2
- Enhances the README with steps for deployment to Azure Storage and Azure App Services

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,31 +2,56 @@
...
-# An Angular single-page application that authenticates users with Azure AD and calls a protected ASP.NET Core web API
+# An Angular single-page application that authenticates users against Azure AD and calls a protected ASP.NET Core web API
+
+ 1. [Overview](#overview)
+ 1. [Scenario](#scenario)
+ 1. [Contents](#contents)
+ 1. [Prerequisites](#prerequisites)
+ 1. [Setup](#setup)
+ 1. [Registration](#registration)
+ 1. [Running the sample](#running-the-sample)
```
Comment (jo-arroyo) [inline README.md:26]:
```suggestion
 1. [Running the sample](#run-the-sample)
```
Link doesn't work otherwise?

**Review by jo-arroyo — APPROVED** [review]
Looks and works great! Instructions are clear and easy to use!

File: README.md (hunk 1 above)
Comment (derisen) [reply README.md]: done!

**Review by idg-sam — APPROVED** [review]
Looks good, works as expected.

**Review by Shama-K — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**